packages/cli-doctor/src/tools/versionRanges.ts

```diff
@@ -3,7 +3,7 @@ export default {
   NODE_JS: '>= 8.3',
   YARN: '>= 1.10.x',
   NPM: '>= 4.x',
-  WATCHMAN: '4.x',
+  WATCHMAN: '>= 4.x',
```

Comment:
4.9.0 was released something like 2 years ago then they transitioned to date ranges that have leading series in the semver parts, which is technically a semver violation but still easily parsed. This let's the range slip up to the new versions, which do work